device/device.go

```diff
@@ -61,8 +64,53 @@ func Flash(typeFlag string) {
 	}
 }
 
+func ListMapping() {
```

Comment:
__[golint]__
* [golint] exported function ListMapping should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->